From 2fdf3c2f17add3877e38f975aacf02f62cc6a510 Mon Sep 17 00:00:00 2001
From: Michael Scott <mike@foundries.io>
Date: Thu, 24 Mar 2022 21:59:59 -0700
Subject: [PATCH] imx8mn-var-som: drop do_reset and
 spl_mmc_get_uboot_raw_sector

u-boot-fio branch uses logic that replaces these functions.

Signed-off-by: Michael Scott <mike@foundries.io>
---
 board/variscite/imx8mn_var_som/spl.c | 25 -------------------------
 1 file changed, 25 deletions(-)

diff --git a/board/variscite/imx8mn_var_som/spl.c b/board/variscite/imx8mn_var_som/spl.c
index 6ff85cdcbf..5f295f0a5d 100644
--- a/board/variscite/imx8mn_var_som/spl.c
+++ b/board/variscite/imx8mn_var_som/spl.c
@@ -269,28 +269,3 @@ void board_init_f(ulong dummy)
 
 	board_init_r(NULL, 0);
 }
-
-int do_reset(cmd_tbl_t *cmdtp, int flag, int argc, char * const argv[])
-{
-	puts("resetting ...\n");
-
-	reset_cpu(WDOG1_BASE_ADDR);
-
-	return 0;
-}
-
-#ifdef CONFIG_SPL_MMC_SUPPORT
-
-#define UBOOT_RAW_SECTOR_OFFSET 0x40
-unsigned long spl_mmc_get_uboot_raw_sector(struct mmc *mmc)
-{
-	u32 boot_dev = spl_boot_device();
-	switch (boot_dev) {
-		case BOOT_DEVICE_MMC1:
-			return CONFIG_SYS_MMCSD_RAW_MODE_U_BOOT_SECTOR;
-		case BOOT_DEVICE_MMC2:
-			return CONFIG_SYS_MMCSD_RAW_MODE_U_BOOT_SECTOR - UBOOT_RAW_SECTOR_OFFSET;
-	}
-	return CONFIG_SYS_MMCSD_RAW_MODE_U_BOOT_SECTOR;
-}
-#endif
-- 
2.17.1

